crates/sqlexec/src/planner/context_builder.rs

```diff
@@ -126,8 +126,8 @@ impl<'a> PartialContextProvider<'a> {
     ) -> Option<Arc<dyn TableFunc>> {
         let catalog = self.ctx.get_session_catalog();
 
-        let resolve_func = |schema, name| {
-            match catalog.resolve_entry(DEFAULT_CATALOG, schema, name) {
+        let resolve_func = |schema: String, name| {
```

Comment:
Could change this into a function to allow something like `fn resolve(schema: impl Into<String>, name: impl Into<String>) -> ...`

Reply:
I'm going to be working on the catalog next, will likely have to refactor some of these methods anyways.